test/e2e/testsuites/specs.go

```diff
@@ -201,12 +202,12 @@ func (pod *PodDetails) SetupStatefulset(client clientset.Interface, namespace *v
 	createdStorageClass := tsc.Create()
 	cleanupFuncs = append(cleanupFuncs, tsc.Cleanup)
 	ginkgo.By("setting up the PVC")
-	tpvc := NewTestPersistentVolumeClaim(client, namespace, volume.ClaimSize, volume.VolumeMode, &createdStorageClass)
+	tpvc := NewTestPersistentVolumeClaim(client, namespace, volume.ClaimSize, volume.VolumeMode, volume.VolumeAccessMode, &createdStorageClass)
 	storageClassName := ""
 	if tpvc.storageClass != nil {
 		storageClassName = tpvc.storageClass.Name
 	}
-	tpvc.requestedPersistentVolumeClaim = generateStatefulSetPVC(tpvc.namespace.Name, storageClassName, tpvc.claimSize, tpvc.volumeMode, tpvc.dataSource)
+	tpvc.requestedPersistentVolumeClaim = generatePVC(tpvc.namespace.Name, storageClassName, tpvc.claimSize, "pvc", tpvc.volumeMode, tpvc.accessMode, tpvc.dataSource)
```

Comment:
Claim size and name ("pvc") order is reversed here.

Reply:
Ah, yes! Thank you!